frontend/src/lib/components/ObjectTags/ObjectTags.tsx

```diff
@@ -17,15 +19,16 @@ type ObjectTagsProps =
     | (ObjectTagsPropsBase & {
           /** Tags CAN'T be added or removed. */
           staticOnly: true
-          onTagSave?: never
-          onTagDelete?: never
+          staticTags?: never
```

Comment:
This `staticTags` concept is a bit awkward for me. If I understand well, we will show on each dashboard an immutable list of all the tags from all the insights that are on the dashboard?

If so, it's a bit... strange? The dashboard itself doesn't really have those tags. Do we also show all of these in the dashboards list? If not, how do we communicate this discrepancy to the users?